### DISCUSSION TITLE
Conversion Module

### DISCUSSION THREAD

**Discussion by ghost** [h3rald/min#90] [Ideas]
A module includes all conversion symbols like `chr`, `ord`, `d2r` etc. However their names specialized by Conversion Rule. Then, somethings will be very easy to find and use and understand ...

Everybody looks at `d2r` and understanding it as a conversion symbol and it must be.

There should be a conversion symbol rule like : `<...>2<...>`
I don't want redefine a symbol `chr` to `chr-int2str` or `ord` to `ord-str2int`.

Am i **false** really? A specified rule would be good. ❓

**Comment by ghost** [comment]
> A module includes all conversion symbols like `chr`, `ord`, `d2r` etc. However their names specialized by Conversion Rule. Then, somethings will be very easy to find and use and understand ...
> 
> Everybody looks at `d2r` and understanding it as a conversion symbol and it must be.
> 
> There should be a conversion symbol rule like : `<...>2<...>`
> I don't want redefine a symbol `chr` to `chr-int2str` or `ord` to `ord-str2int`.
> 
> Am i **false** really? A specified rule would be good. ❓

Hehehehe... I know a solution for this: an IDE which supports **documentation**. ✅ ❓

**Reply by ghost** [reply]
In IDE, there is no need to explify parameters and returnings at symbol. IDE automaticly says what it is.

**Reply by h3rald** [reply]
Hmmmm ok, I understand what you are suggesting, but hat would require some new way to lookup those symbols...

instead, you can do something similar with user-defined sigils! The only difference is that they have to be placed in front of strings, for example:

```
47 conv"chr-int->str"
```
